python_scripts/linear_models_sol_04.py

```diff
@@ -131,7 +131,7 @@
 
 # %% [markdown] tags=["solution"]
 # ```{tip}
-# We always advise to use l2-penalized model instead of non-penalized model
+# We advise to always use penalized models instead of non-penalized models
```

Comment:
I don't think this is the right message now. We exactly want to advise using a l2 penalty. E.g. we don't want people to use a l1 penalty. So we can make it explicit (as in my previous quiz comment) what type of penalty (shrink weight magnitude towards zeros) and add it explicitly that this is called l2 penalty.